config/history.go

```diff
@@ -129,7 +129,6 @@ var DefaultChains = map[common.ChainName]*History{
 	common.ChainNameTestnet: {
 		Records: []*Record{
 			// TODO: coalesce compatible records
-			// TODO: rename archives to match compatible API
```

Comment:
lol k :shrug:

Reply:
I didn't want to name the most recent testnet "damask" because with testnet, we'll probably (if we ever want to support the whole testnet history) end up having more than one archive that can be accessed with DamaskConsensusApiLite. Then we'd need "damask" as a map key for 2+ archives.

Reply:
o ok